src/datanode/src/sql/alter.rs

```diff
@@ -133,4 +143,25 @@ mod tests {
             _ => unreachable!(),
         }
     }
+
+    #[tokio::test]
+    async fn test_alter_to_request_with_renaming_table() {
```

Comment:
If we test parsing the SQL in `alter_parser.rs`, then we only need to check `alter_to_request()` returns the expected error.